xapian-core/tests/harness/backendmanager.cc

```diff
@@ -207,6 +259,13 @@ BackendManager::get_writable_database(const string &, const string &)
     invalid_operation("Attempted to open a disabled database");
 }
 
+Xapian::WritableDatabase
+BackendManager::get_alt_writable_database(const string &, const string &)
```

Comment:
Putting the `&` with the type seems to be widely preferred as C++ style and we've decided to follow that (a lot of existing Xapian code doesn't conform but please follow this for new code so the problem at least doesn't get worse.  Same for pointer types (`char*` not `char *`).  Currently `xapian-check-patch` doesn't try to check for this.

Reply:
> Currently `xapian-check-patch` doesn't try to check for this.

FYI, added a check for this (at least in common cases) yesterday.